src/runner.rs

```diff
@@ -1259,7 +1179,7 @@ struct FunctionContext {
 }
 
 impl FunctionContext {
-	pub fn new(function: FuncRef) -> Self {
+	pub(crate) fn new(function: FuncRef) -> Self {
```

Comment:
`pub(crate)` is usually a code smell, it shouldn't be used apart from for internal functions that you want to expose to tests. This can be `pub` or you can rearrange the module hierarchy to allow you to make it private but accessible from the right places. Probably the former.

Reply:
Changed from `pub(crate)` to `pub`.